assets/js/translator.js

```diff
@@ -111,6 +116,11 @@ if(modeEnabled('translation')) {
                     $('#fileInput').show();
                     $('div#fileName').hide();
                     $('div#docTranslation').fadeIn('fast');
+                    if($('#detect').hasClass('active') === true) {
+                        $('#srcLang1').addClass('active');
```

Comment:
I’m somewhat afraid of doing things in a more ad hoc manner like this because just changing the classes around might not change some of the program state that should also have been changed.

`$('#srcLang1').click()` might be better? and when switching back, do some of the things in `handleDetectLanguageSuccessResponse` if detect was selected?

Reply:
@sushain97
When you say "do some of the things in `handleDetectLanguageSuccessResponse` if detect was selected", do you mean move the stuff that is after $('#detect').hasClass('active') === true?